internal/pkg/manifest/workload.go

```diff
@@ -764,6 +780,10 @@ func validateAdvancedPlatform(platform PlatformArgs) error {
 	return fmt.Errorf("platform pair %s is invalid: fields ('osfamily', 'architecture') must be one of %s", platform.String(), prettyValidPlatforms)
 }
 
+func isWindowsPlatform(platform *PlatformArgsOrString) bool {
+	return platform.OS() == OSWindows || platform.OS() == OSWindowsServer2019Core || platform.OS() == OSWindowsServer2019Full
```

Comment:
nit: I usually create a slice and loop over it so that in the future if we add a new windows os family we don't have to update this method:
```go
var windowsOSFamilies = []string{OSWindows, OSWindowsServer2019Core, OSWindowsServer2019Full}

func isWindowsPlatform(platform *PlatformArgsOrString) bool {
   for _, win := range windowsOSFamilies {
      if platform.OS() == win { return true }
   }
   return false
}
```